ts/image-occlusion/generate.ts

```diff
@@ -40,22 +37,34 @@ export function generate(hideInactive: boolean): { occlusionCloze: string; noteC
         const obJson = object.toJSON();
         noteCount++;
         if (obJson.type === "group") {
-            clozeData += getGroupCloze(object, index, hideInactive);
+            clozeData += getGroupCloze(canvas, object, index, hideInactive);
         } else {
-            clozeData += getCloze(object, index, null, hideInactive);
+            clozeData += getCloze(canvas, object, index, null, hideInactive);
         }
     });
 
     occlusionCloze += clozeData;
     return { occlusionCloze, noteCount };
 }
 
-const getCloze = (object, index, relativePos, hideInactive): string => {
+function floatToDisplay(number: number): string {
+    return number.toFixed(6).replace(/^0+|0+$/, "");
```

Comment:
The 0-tail-trimming did not seem to work for me here, but switching the RegEx to global seemed to do the trick

```suggestion
    return number.toFixed(6).replace(/^0+|0+$/g, "");
```

Reply:
Nice catch - coming from a Python background, JS's regex handling always trips me pu.